CHANGELOG.md

```diff
@@ -9,10 +9,56 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum
 
+- Changed storage layout (#1293)
```

Comment:
@pavel-mukhanov I suppose this related block to MerkleDB should be moved under **exonum-merkledb** section